data/padics/padic_integers.lean

```diff
@@ -13,7 +13,7 @@ open nat padic
 noncomputable theory
 local attribute [instance] classical.prop_decidable
 
-def padic_int (p : ℕ) [p.prime] := {x : ℚ_[p] // ∥x∥ ≤ 1}
+def padic_int (p : ℕ) [nat.prime p] := {x : ℚ_[p] // ∥x∥ ≤ 1}
```

Comment:
why?

Reply:
There is a conflict with the global `prime` now, since `nat` is open here